notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb

```diff
@@ -6,13 +6,13 @@
    "metadata": {},
```

Comment:
<p><s>The optimization process contains the following steps:</s></p><p>sentence not bind with the next text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>